google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/JsonStreamWriterTest.java

```diff
@@ -518,9 +520,10 @@ public void testWithoutIgnoreUnknownFields() throws Exception {
       try {
         ApiFuture<AppendRowsResponse> appendFuture = writer.append(jsonArr);
         Assert.fail("expected ExecutionException");
-      } catch (Exception ex) {
+      } catch (AppendSerializtionError ex) {
```

Comment:
nit: Also assert that ex.getStreamName() returns TEST_STREAM ?

Reply:
done